mkosi.md

```diff
@@ -300,6 +301,14 @@ they should be specified with a boolean argument: either "1", "yes", or "true" t
 | `ImageVersion=` | no    | yes              | match fails             |
 | `Bootable=`     | no    | no               | match auto feature      |
 
+### [Preset] Section
+
+`Depends=`, `--depends=`
```

Comment:
Wouldn't it be better to call this After? "Depends" doesn't sound good because "on" is missing, i.e we say "a depends on b", not "a depends b".

Reply:
Or "Dependencies". But I think "After" is nicer.

Reply:
Went for `Dependencies=`